### PR TITLE
feat - wishlist screen

### PR DESCRIPTION
**Pull request by prachi-sahani**
Wishlist screen display the products wishlisted by the user
(Note - styling for the product cards is coming from styles.css which is common for all the product cards)

### Demo
ecom - wishlist page

### CODE REVIEW

File: css/wishlist-page.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  color: var(--black-600);
+}
+
+.product-card{
+    max-width: 21%;
+}
+
+.product-card .btn-close{
+    top: 0.25rem;
+    right: 0.25rem;
```
Comment (Hamdrive) [inline css/wishlist-page.css]: Use `px` instead

**Review by Hamdrive — APPROVED** [review]
Good work done, will suggest to consider changing units as mentioned. Else code is good to merge!

**Review by abhij1607 — APPROVED** [review]
Clean Code, Looks good to merge

File: css/wishlist-page.css (hunk 1 above)
Comment (prachi-sahani) [reply css/wishlist-page.css]: done!

**Pull request merged**